docs/src/index.md

```diff
@@ -6,8 +6,8 @@ CurrentModule = HydroPowerSimulations
 
 ## Overview
 
-`HydroPowerSimulations.jl` is a [`Julia`](http://www.julialang.org) package that provides blah blah
+`HydroPowerSimulations.jl` is a [`Julia`](http://www.julialang.org) package that extends [`PowerSimulations.jl`](https://github.com/NREL-Sienna/PowerSimulations.jl) for modeling of hydro generation technology in production cost modeling simulations.
```

Comment:
```suggestion
`HydroPowerSimulations.jl` is a [`Julia`](http://www.julialang.org) package that extends [`PowerSimulations.jl`](https://nrel-sienna.github.io/PowerSimulations.jl/stable/) for modeling of hydro generation technology in production cost modeling simulations.
```

Reply:
`production cost modeling simulations` -> `operational simulations`

Reply:
@rodrigomha, can you update the hyperlink above, too?

Reply:
Miss that one